content/en/docs/setup/independent/high-availability.md

```diff
@@ -10,513 +10,217 @@ content_template: templates/task
+After this setup you should have these files on the first bootstrapped member.
+You will need to put these files on all of your control-plane nodes.
 
-1. Run the following:
-     ```bash
-     mkdir -p /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/ca.pem /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client.pem /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client-key.pem /etc/kubernetes/pki/etcd
-     ```
+1. `/etc/kubernetes/pki/etcd/ca.crt`
+1. `/etc/kubernetes/pki/apiserver-etcd-client.crt`
+1. `/etc/kubernetes/pki/apiserver-etcd-client.key`
 
-## Run `kubeadm init` on `master0` {#kubeadm-init-master0}
+Here is one way to do it, but there are many reasons why this might not work for
+you. For instance, you may be running any other linux distro.
 
-1. In order for kubeadm to run, you first need to write a configuration file:
-     ```bash
-     cat >config.yaml <<EOF
-     apiVersion: kubeadm.k8s.io/v1alpha1
-     kind: MasterConfiguration
-     api:
-      advertiseAddress: <private-ip>
-     etcd:
-      endpoints:
-      - https://<etcd0-ip-address>:2379
-      - https://<etcd1-ip-address>:2379
-      - https://<etcd2-ip-address>:2379
-      caFile: /etc/kubernetes/pki/etcd/ca.pem
-      certFile: /etc/kubernetes/pki/etcd/client.pem
-      keyFile: /etc/kubernetes/pki/etcd/client-key.pem
-     networking:
-      podSubnet: <podCIDR>
-     apiServerCertSANs:
-     - <load-balancer-ip>
-     apiServerExtraArgs:
-      apiserver-count: "3"
-     EOF
-     ```
+```
+scp /etc/kubernetes/pki/etcd/ca.crt ubuntu@ETCD_HOST_1:/home/ubuntu
+scp /etc/kubernetes/pki/apiserver-etcd-client.crt ubuntu@ETCD_HOST_1:/home/ubuntu
+scp /etc/kubernetes/pki/apiserver-etcd-client.key ubuntu@ETCD_HOST_2:/home/ubuntu
+```
 
-    Ensure that the following placeholders are replaced:
+## Setting up the control plane
+
+### Creating a load balancer
+
+In a cloud environment you want to put your control planes behind a tcp
+forwarding load balancer. This load balancer will distribute traffic to all
+healthy control plane nodes in its target list. The health check to determine if
+an api server is healthy or not is a TCP check on the port the kube-apiserver is
+listening on (default is `:6443`).
+
+The load balancer must have a resolvable DNS name (we do not suggest using an IP
+address directly in a cloud environment). The load balancer must have the ability to
+communicate with all control plane nodes on the apiserver port and it must allow
+incoming traffic on its listening port.
+
+After this step, you have a resolvable DNS name for your load balancer.
+
+You should add the control plane nodes to the load balancer but be aware they
+will fail the healthcheck until the kube-apiserver is running.
+
+### The first control plane node
+
+#### Prerequisites
+
+1. This node must have [kubeadm, kubelet and kubectl][installation] installed.
```

Comment:
I'd link and list individually:

This node must have installed 
 
* [kubeadm]()
* [kubelet]()
* [kubectl]()